english/set_up_a_small_goal/setup_a_small_goal.md

```diff
@@ -0,0 +1 @@
+before you do big thing, we should make process step by step. Let's finish a small goal first, for example, making 100million dollars.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.